.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
```

Comment:
Shouldn't this config be part of `pyproject.toml`?

Reply:
unfortunately flake 8 does not support pyproject.toml

> https://github.com/PyCQA/flake8/issues/234

the same holds for coverage